stats.go

```diff
@@ -21,26 +21,27 @@ func getInitStats() *types.Statistics {
 		FIFO:          getInputNewMap("fifo"),
 		GRPC:          getInputNewMap("grpc"),
 		Falco:         expvar.NewMap("falco.priority"),
-		Slack:         getOutputNewMap("slack"),
-		Rocketchat:    getOutputNewMap("rocketchat"),
-		Mattermost:    getOutputNewMap("mattermost"),
-		Teams:         getOutputNewMap("teams"),
-		Datadog:       getOutputNewMap("datadog"),
-		Discord:       getOutputNewMap("discord"),
-		Alertmanager:  getOutputNewMap("alertmanager"),
-		Elasticsearch: getOutputNewMap("elasticsearch"),
-		Loki:          getOutputNewMap("loki"),
-		Nats:          getOutputNewMap("nats"),
-		Influxdb:      getOutputNewMap("influxdb"),
-		AWSLambda:     getOutputNewMap("awslambda"),
-		AWSSQS:        getOutputNewMap("awssqs"),
-		AWSSNS:        getOutputNewMap("awssns"),
-		SMTP:          getOutputNewMap("smtp"),
-		Opsgenie:      getOutputNewMap("opsgenie"),
-		Statsd:        getOutputNewMap("statsd"),
-		Dogstatsd:     getOutputNewMap("dogstatsd"),
-		Webhook:       getOutputNewMap("webhook"),
-		AzureEventHub: getOutputNewMap("azureeventhub"),
+		Slack:         expvar.NewMap("outputs.slack"),
+		Rocketchat:    expvar.NewMap("outputs.rocketchat"),
+		Mattermost:    expvar.NewMap("outputs.mattermost"),
+		Teams:         expvar.NewMap("outputs.teams"),
+		Datadog:       expvar.NewMap("outputs.datadog"),
+		Discord:       expvar.NewMap("outputs.discord"),
+		Alertmanager:  expvar.NewMap("outputs.alertmanager"),
+		Elasticsearch: expvar.NewMap("outputs.elasticsearch"),
+		Loki:          expvar.NewMap("outputs.loki"),
+		Nats:          expvar.NewMap("outputs.nats"),
+		Influxdb:      expvar.NewMap("outputs.influxdb"),
+		AWSLambda:     expvar.NewMap("outputs.awslambda"),
+		AWSSQS:        expvar.NewMap("outputs.awssqs"),
+		AWSSNS:        expvar.NewMap("outputs.awssns"),
+		SMTP:          expvar.NewMap("outputs.smtp"),
+		Opsgenie:      expvar.NewMap("outputs.opsgenie"),
+		Statsd:        expvar.NewMap("outputs.statsd"),
+		Dogstatsd:     expvar.NewMap("outputs.dogstatsd"),
+		Webhook:       expvar.NewMap("outputs.webhook"),
+		AzureEventHub: expvar.NewMap("outputs.azureeventhub"),
+		GCPPubSub:     expvar.NewMap("outputs.gcppubsub"),
```

Comment:
this whole block has been changed in last commit by a new method for declaring expvar, please rebase on upstream master

Reply:
This has been corrected.